app/views/user_mailer/_returned_overdue.html.erb

```diff
@@ -0,0 +1,4 @@
+<p>This is a notification for your late equipment checkin. </p>
+
+<p>Your late fee is <%= number_to_currency(@reservation.equipment_model.late_fee) %> per day, for a total of <%= number_to_currency(((@reservation.checked_in.to_date - @reservation.due_date.to_date )).to_i*(@reservation.equipment_model.late_fee))%>
```

Comment:
take out the `to_date` on `due_date`, but not on `checked_in` since that's still a `DateTime`.